src/libc-wrappers/libc-wrappers.c

```diff
@@ -15,7 +15,153 @@
  */
 
 
+#include <pthread.h>
 #include <signal.h>
+#include <link.h>
+
+
+/*
+ * __libc_start_main
```

Comment:
It might be good to handle `__libc_start_main` in a separate commit, because the corresponding [glibc change](https://sourceware.org/git/?p=glibc.git;a=commit;h=035c012e32c11e84d64905efaf55e74f704d3668) is very different from that of the other `pthread_*` symbols, which were moved around as part of the [libpthread removal](https://sourceware.org/ml/libc-alpha/2019-10/msg00080.html) project.

Reply:
I took the liberty to update the branch.